AAI/AAIConnectProfile.md

```diff
@@ -99,12 +114,20 @@ described herein, encoded via JWS Compact Serialization per
 `ga4gh_passport_v1` as a space-separated entry within the `scope` claim but
```

Comment:
I think we may drop the requirement that the access token must be a JWT, it can just have `ga4gh_passport_v1` among its scopes as defined in [OAuth 2.0 RFC section 3.3](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3)